README.md

```diff
@@ -86,33 +68,41 @@ details. In addition Kubernetes itself doesn't yet support sub-resources
 for custom resource definitions. As a result the provider has to extract
 the SPICE proxy URL from the configuration of the `spice-proxy` service.
 
-== Notes
+## Notes
 
-=== How to get the default token from Kubernetes
+### How to get the default token from Kubernetes
 
 List the set of secrets:
 
+  ```
   $ kubectl get secrets
   NAME                  TYPE                                  DATA      AGE
   default-token-7psxt   kubernetes.io/service-account-token   3         20d
+  ```
 
 Get the details of the `default` token:
 
+  ```
```

Comment:
might not be part of this PR, but we can simplify the steps below by using:

`kubectl describe secret default-token-7psxt`

So decoding the token is not required.